include/chemfiles/files/XDRFile.hpp

```diff
@@ -19,6 +19,10 @@ class XDRFile final : public BigEndianFile {
     XDRFile(std::string path, File::Mode mode);
     ~XDRFile() = default;
 
+    using BinaryFile::read_f32;
+    using BinaryFile::read_f64;
+    using BinaryFile::write_f32;
```

Comment:
Not sure if this is the best way to expose the base functions in the derived class.

Reply:
That's how I would do it as well =)